lib/mapper.js

```diff
@@ -47,6 +48,129 @@ exports.track = function(track){
 };
 
 /**
+ * Map Completed Order
+ *
+ * Docs: http://docs.klaviyo.com/article/60-ecommerce-integrating-a-custom-cart
+ *
+ * Must call Top level request for "Placed Order" and separate request for each item in products array
+ * @param {Track} track
+ * @return {Oject}
+ * @api private
+ */
+
+exports.completedOrder = function(track, settings){
+  var products = track.products();
+  var categories = formatCategories(products);
+  var productNames = formatNames(products);
+  var items = formatItems(products);
+
+  // TODO return two payloads
+  var payloads = {};
+
+  // Placed Order payload
```

Comment:
This comment doesn't actually tell us much about this code. I'd remove it

If you feel the need for a comment you should generally ask yourself if you're describing _what_ a piece of code is doing (usually at this point, you should see if you can refactor your code to be more clear) or if you're describing _why_ something is happening (super useful comment)